applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -137,16 +139,24 @@ def _ComputeLiftFromJumpCondition(self):
     def _ComputeLiftFromJumpCondition3D(self):
 
         potential_integral = 0.0
+        drag_integral = 0.0
         for cond in self.trailing_edge_model_part.Conditions:
             length = cond.GetGeometry().Area()
             for node in cond.GetNodes():
                 potential = node.GetSolutionStepValue(CPFApp.VELOCITY_POTENTIAL)
                 auxiliary_potential = node.GetSolutionStepValue(CPFApp.AUXILIARY_VELOCITY_POTENTIAL)
+                velocity = node.GetValue(KratosMultiphysics.VELOCITY)
+                velocity_normal_component = _DotProduct(self.wake_normal,velocity)
                 potential_jump = potential - auxiliary_potential
                 potential_integral += 0.5 * length * potential_jump
+                drag_integral -= 0.5 * length * potential_jump * velocity_normal_component
 
+        free_stream_velocity = self.fluid_model_part.ProcessInfo.GetValue(CPFApp.FREE_STREAM_VELOCITY)
+        free_stream_velocity_norm2 = _DotProduct(free_stream_velocity,free_stream_velocity)
```

Comment:
```suggestion
        free_stream_velocity_norm2 = free_stream_velocity.norm_2()
```
Here we could use the `norm_2` method right?